superset-frontend/plugins/legacy-plugin-chart-world-map/src/WorldMap.js

```diff
@@ -187,6 +192,10 @@ function WorldMap(element, props) {
           col: entity,
           op: '==',
           val,
+          formattedVal:
+            getColumnLabel(entity) === formData.granularitySqla
+              ? String(timestampFormatter(val))
+              : String(val),
```

Comment:
While this will probably work well in legacy charts that still have a time section, this won't work for "modern" charts that don't have a time section when `GENERIC_CHART_AXES` is enabled. For this chart this might be the correct solution, as I don't remember if `viz.py` returns `colnames` (=all columns) and `coltypes` (=the respective datatypes for said columns), but at least for charts that are powered by the v1 chart API we should check if the datatype of the column is `GenericDataType.TEMPORAL`, and if so, format it with `timestampFormatter`.

Reply:
Tested with Pie Chart with GENERIC_CHART_AXES enabled, can confirm that it doesn't work as expected :(